package.json

```diff
@@ -31,6 +31,7 @@
     "moment": "2.29.1",
     "obj-str": "1.1.0",
     "preact": "10.5.15",
+    "protobuf-compiler": "1.0.2",
```

Comment:
@JakeSquier what is the protobuf-compiler for?

Reply:
Ooh I apologize I had added that on accident. Uninstalling and pushing up updated branch for ya!

Reply:
appreciate it. is the `bazel.gpg` file also accidental?

Reply:
Yes, I had installed Proto-buf and Bazel.gpg incorrectly when I was following the end to end guide I assume. I'll take care of it real quick I apologize for the inconvenience.